### DISCUSSION TITLE
PRINTING Howto install cups:  WSL2 withh openSUSE-Leap15.2

### DISCUSSION THREAD

**Discussion by F-Haferkorn** [microsoft/WSL#5940] [General]
After some tweaks I got openSUSE-Leap-15.2 in WSL2 up and running.
The KDE Plasma is coming up an seems to work, as browser and audio is OK.

HOW can I **use A Printer** from that WSL2 system?

I tried hp-toolbox, which relies on hp-systray, which itself requires cups.
So the question is how do I get cups up and running ?

!I will need a handover on  the Windows side! 

```
$cat /etc/os-release
NAME="openSUSE Leap"
VERSION="15.2"
ID="opensuse-leap"
ID_LIKE="suse opensuse"
VERSION_ID="15.2"
PRETTY_NAME="openSUSE Leap 15.2"
ANSI_COLOR="0;32"
CPE_NAME="cpe:/o:opensuse:leap:15.2"
BUG_REPORT_URL="https://bugs.opensuse.org"
HOME_URL="https://www.opensuse.org/"
```
and
```
$uname -a
Linux ALICE 4.19.128-microsoft-standard #1 SMP Tue Jun 23 12:58:10 UTC 2020 x86_64 x86_64 x86_64 GNU/Linux

```

**Comment by therealkenc** [comment]
Same #5621. No repro offered on "hp-toolbox", but it if uses libusb, that's #2195.

Something something set up a shared printer with SMB/CIFS something (maybe)...
